typing/btype.mli

```diff
@@ -82,18 +82,10 @@ val commu_repr: commutable -> commutable
 
 (**** polymorphic variants ****)
 
-val row_repr: row_desc -> row_desc
-        (* Return the canonical representative of a row description *)
-val row_field_repr: row_field -> row_field
-val row_field: label -> row_desc -> row_field
-        (* Return the canonical representative of a row field *)
-val row_more: row_desc -> type_expr
-        (* Return the extension variable of the row *)
-
 val is_fixed: row_desc -> bool
 (* Return whether the row is directly marked as fixed or not *)
 
-val row_fixed: row_desc -> bool
+val has_fixed_explanation: row_desc -> bool
 (* Return whether the row should be treated as fixed or not.
    In particular, [is_fixed row] implies [row_fixed row].
```

Comment:
The comment should be updated to at least no longer refer to the former function name `row_fixed`.

Reply:
Done